Documentation/prereqs.md

```diff
@@ -0,0 +1,51 @@
+.NET Core native prerequisites
+==============================
+
+This document outlines the dependencies needed to run .NET Core tools and applications. 
+
+## Windows dependencies
+On Windows, the only dependency is the VC++ Redistributable. Depending on the version of Windows you are running on, the versions are changing.
+
+> **Note:** these dependencies are chained in the installer, you only need to install them manually if you are using
+> the installer script (`dotnet-install.sh` or `dotnet-install.ps1`). 
+
+* Windows 10
+    * [Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
+* Pre-Windows 10
+    * Please make sure that your Windows installation is up-to-date and includes hotfix [KB2533623] (https://support.microsoft.com/en-us/kb/2533623) installed through Windows Update.
+    * [Visual C++ Redistributable for Visual Studio 2012 Update 4](https://www.microsoft.com/en-us/download/confirmation.aspx?id=30679)
+
+## Ubuntu
+Ubuntu distributions require the following libraries installed:
+
+- libunwind8 
+- libunwind8-dev
+- gettext
+- libicu-dev
+- liblttng-ust-dev
+- libcurl4-openssl-dev
+- libssl-dev
+- uuid-dev
+- unzip
+
+
+## CentOS
+CentOS distributions require the following libraries installed:
+
+* deltarpm
+* epel-release
+* unzip
+* libunwind
+* gettext 
+* libcurl-devel 
+* openssl-devel 
+* zlib 
+* libicu-devel
+
+## OS X 
+OS X requires the following libraries and versions installed:
+
+* libssl >= 1.1
```

Comment:
>= 1.0.1.

The current version from homebrew is 1.0.2, and 1.1 isn't quite released yet, and may not even be compatible.